src/nvim/strings.c

```diff
@@ -220,6 +225,9 @@ char_u *vim_strsave_shellescape(const char_u *string,
       ++length;                         /* insert backslash */
       p += l - 1;
     }
+    if (*p == '\\' && fish_like) {
+      length++; // insert backslash
```

Comment:
```suggestion
      length++;  // insert backslash
```

(2 spaces before a comment)

Reply:
Whoops, I'm on a different computer and couldn't get `make lint` actually running, so I missed this one. Fixed now.